app/lib/features/cross_signing/views/sas_cancelled.dart

```diff
@@ -43,7 +43,7 @@ class SasCancelledView extends StatelessWidget {
           SizedBox(
             width: MediaQuery.of(context).size.width * 0.40,
             child: ElevatedButton(
-              child: const Text('Got it'),
+              child: Text(L10n.of(context).sasGotIt),
```

Comment:
Prefix `sas` stands for?

Reply:
`sas` stands for short authentication string.
Event handling of verification consists of request stage and sas stage.
Request stage stands for stage prior to request accept.
Sas stage stands for stage after request accepted.
There are many sentences related to verification, and I split them into request group and sas group.